src/app/view/components/_common/dot-page-selector/dot-page-selector.component.ts

```diff
@@ -179,4 +195,13 @@ export class DotPageSelectorComponent implements ControlValueAccessor {
     private autoSelectUniqueResult(): void {
         this.onSelect(this.results.data[0]);
     }
+
+    private cleanInput(event: any): string {
+        event.originalEvent.target.value = this.cleanQuery(event.query);
+        return event.originalEvent.target.value;
+    }
+
+    private cleanQuery(query: string): string {
+        return !VALID_INPUT_REGEX.test(query) ? query.replace(REPLACE_INPUT_REGEX, '') : query;
```

Comment:
What is this validating? I think the regex could have a better naming like: "NO_SPECIAL_CHAR" or "VALID_URL" or something like that.